app/templates/_view_agreements_results.html

```diff
@@ -0,0 +1,12 @@
+{% for supplier_framework in supplier_frameworks %}
+<div class="search-result">
+  <h2 class="search-result-title">
+    <a href="{{ url_for('.view_signed_agreement', supplier_id=supplier_framework.supplierId, framework_slug=supplier_framework.frameworkSlug) }}">{{ supplier_framework.supplierName }}</a>
+  </h2>
+  <ul class="search-result-important-metadata">
+    <li class="search-result-metadata-item">
+      Submitted: {{ supplier_framework.agreementReturnedAt }}
```

Comment:
Ah yep, I see it now. Most other places we tend to do the formatting in the template I think, not sure why this got done in the view.